code401Challenges/src/test/java/code401Challenges/hashtable/TreeIntersectionTest.java

```diff
@@ -1,7 +1,114 @@
 package code401Challenges.hashtable;
 
+import code401Challenges.tree.Node;
+import org.junit.Test;
 import static org.junit.Assert.*;
 
+import java.util.HashSet;
+
 public class TreeIntersectionTest {
 
-}
\ No newline at end of file
+    @Test
+    public void treeIntersectionTest1 () {
+        //              10
+        //             /  \
+        //            5    15
+        //           / \   / \
+        //          2  7  12  17
+
+
+        Node root1 = new Node(10,
+                new Node(5, new Node(2), new Node(7)),
+                new Node(15, new Node(12), new Node(17)));
+
+        //            5
+        //           / \
+        //          0  2
+
+        Node root2 = new Node(5,
+                new Node(0), new Node(2));
+
+        HashSet<Integer> expected = new HashSet<>();
+        expected.add(2);
+        expected.add(5);
+
+        System.out.println(TreeIntersection.treeIntersection(root1, root2));
```

Comment:
Now you want to change the println statements into assert statements. In this case you can just use `.equals()` on expected with what you get back from calling your method.  That will look like: `assertTrue(expected.equals(TreeIntersection.treeIntersection(root1, root2)));`